pkg/handlers/system_intake.go

```diff
@@ -203,3 +205,122 @@ func (h SystemIntakeHandler) Handle() http.HandlerFunc {
 		}
 	}
 }
+
+// NewSystemIntakeLifecycleIDHandler is a constructor for how we handle
+// LifecycleID assignment
+func NewSystemIntakeLifecycleIDHandler(
+	base HandlerBase,
+	assign func(context.Context, *models.SystemIntake) error,
+) SystemIntakeLifecycleIDHandler {
+	return SystemIntakeLifecycleIDHandler{
+		HandlerBase:       base,
+		AssignLifecycleID: assign,
+	}
+}
+
+// SystemIntakeLifecycleIDHandler is the handler for assigning
+// a lifecycleID to a SystemIntake
+type SystemIntakeLifecycleIDHandler struct {
+	HandlerBase
+	AssignLifecycleID func(context.Context, *models.SystemIntake) error
+}
+
+type lcidFields struct {
+	LCID      *string `json:"lcid"`
+	ExpiresAt string  `json:"lcidExpiresAt"`
+	Scope     string  `json:"lcidScope"`
+	NextSteps string  `json:"lcidNextSteps"`
+}
```

Comment:
Created a small type rather than using the larger `models.SystemIntake` for a couple of reasons:
- won't accidentally pull in fields other than the 4 we are concerned with
- parsing the date needed some special handling